src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 230;
+  }
+
+  if (days > 3 && days <= 6) {
+    return 220;
+  }
+
+  if (days >= 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The return value '80' is incorrect. It should be the daily cost times the number of days without any discount when renting for less than 3 days. Use a constant for the daily cost and calculate accordingly.